src/frontend/shortlist/src/components/layout/shortlists/ShortlistRow.vue

```diff
@@ -16,17 +21,43 @@ export default {
     itemDragDrop(e) {
       this.$emit("itemDragDrop", e);
     },
+    switchColor(colorChoice) {
+      localStorage.setItem('color-options', colorChoice); 
+      this.currentColor = localStorage.getItem('color-options')
+  }
   },
 };
 </script>
 
 <template>
   <div
-    class="layout-list-row"
-    :style="{ backgroundColor: listSettings.color }"
```

Comment:
This is how color is obtained originally from the .js file, which is also the part we need to make the change to affect user profile, would get local works the same?

Reply:
> This is how color is obtained originally from the .js file, which is also the part we need to make the change to affect user profile, would get local works the same?

I think I had to change listSettings.color, otherwise it would be the same color always? I may be missing something here.

Reply:
You do need to change listSettings.color though a new assignment. Not deleting the whole statement using something else, or it wouldn't render the color stored in db